src/main/java/com/foo/gosucatcher/domain/member/application/dto/request/MemberSignUpRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.foo.gosucatcher.domain.member.application.dto.request;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+import com.foo.gosucatcher.domain.member.domain.Member;
+
+public record MemberSignUpRequest(
+	@NotEmpty @NotBlank @Min(2) @Max(20) String name,
```

Comment:
```suggestion
	@Size(min = 2, max = 20 message = "message") String name
```
@Min, @Max 둘다 쓰는 것 보다 @Size 하나로 쓰고 그 안에서 최솟값, 최대값 지정 가능합니다!

Reply:
위쪽 리뷰에서도 비슷한 방식으로 @Length와@NotEmpty를 쓰는 것을 말씀해주셨는데,
일단 적용하기도 했고 ㅠ 디테일한 예외 메시지를 위해 기존 방식을 쓰도록 하겠습니다...!
나중에 유효성 검증할 때, 이 방식도 적용해야겠네요!! 알려주셔서 감사합니다👍